assembl/static2/js/app/components/debate/multiColumns/columnsView.jsx

```diff
@@ -14,21 +16,15 @@ class ColumnsView extends React.Component {
   shouldShowTabs = columnsCount => columnsCount * MIN_WIDTH_COLUMN > Math.min(this.props.screenWidth, APP_CONTAINER_MAX_WIDTH);
 
   render = () => {
-    const { messageColumns: columns, identifier, debateData } = this.props;
+    const { messageColumns: columns } = this.props;
     if (!Array.isArray(columns)) return null;
-    const showSynthesis = columns.some(column => !!column.header);
+    const showSynthesis = columns.some(column => !!get(column, 'columnSynthesis.body'));
     return (
       <div className="max-container">
         {this.shouldShowTabs(columns.length) ? (
-          <TabbedColumns {...this.props} showSynthesis={showSynthesis} identifier={identifier} debateData={debateData} />
+          <TabbedColumns {...this.props} showSynthesis={showSynthesis} />
         ) : (
-          <MultiColumns
-            {...this.props}
-            width={`${100 / columns.length}%`}
-            showSynthesis={showSynthesis}
-            identifier={identifier}
-            debateData={debateData}
-          />
+          <MultiColumns {...this.props} width={`${100 / columns.length}%`} showSynthesis={showSynthesis} />
```

Comment:
`width` as a string - can't it be an integer and then typecasted at the lowest-level `style` parameter?

Reply:
this one wasn't part of this commit but I think it is ok to pass a specific `width` prop since we are managing columns here (and it's a percentage).